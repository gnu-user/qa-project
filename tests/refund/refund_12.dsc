TITLE
    Refund Transfer Credit Operation (Value Overflow)
DESCRIPTION
    Run the login transaction using an admin account. Initiate the refund 
    transaction. When prompted, enter a buyer which exists. When prompted, 
    enter a seller which exists. When prompted, enter a valid decimal value for 
    the refund amount (which is greater than the maximum credit value, of 
    999,999). Verify that the value is rejected, and an error occurs.
DEPENDENCIES
    refund_7
