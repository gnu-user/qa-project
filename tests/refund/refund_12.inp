login
administrator
refund
buyer
seller
1000000.00