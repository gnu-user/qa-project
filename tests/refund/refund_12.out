Welcome.
Current user accounts file read successfully.
Available tickets file read successfully.
> 
Enter username: 
Username accepted.
> 
Enter the buyer's username: 
Enter the seller's username: 
Enter the amount of credit to transfer: 
Invalid credit value, amount cannot exceed 1000.00.
> 